src/basic/unit-def.c

```diff
@@ -169,6 +169,7 @@ DEFINE_STRING_TABLE_LOOKUP(path_state, PathState);
 
 static const char* const scope_state_table[_SCOPE_STATE_MAX] = {
         [SCOPE_DEAD]         = "dead",
+        [SCOPE_CHOWN_CGROUP] = "chown-cgroup",
```

Comment:
event though this is strictly about cgroups, I'd probably call this more generically "chown". I also would say "start" in the name, to make clear this is about starting the scope. Hence maybe call this SCOPE_START_CHOWN. 

This also has the nice effect that the state is similarly called to SOCKET_START_CHOWN of socket units. There's its about chowning the socket inode, instead of the cgroup, but if we'd add support for more stuff to chown, then it would also subsumed in SOCKET_START_CHOWN there, so having a generic name is good.

but yeah, this is pure bikeshedding